testdata/TestCheckAgainstSpecData.json

```diff
@@ -7,16 +7,16 @@
         "hash": 1,
```

Comment:
Curious: how did you re-generate this test data?

Reply:
+1, were they changed by hand?

Reply:
yup! The only way I managed was to pull printed proofs from ibc-go tests :sweat_smile: That's why I added #348

Reply:
Ah yea, figured as much when I saw the issue! 😅 sometimes caveman approach = best approach